src/components/Comments.jsx

```diff
@@ -33,10 +35,17 @@ const Comments = ({ article_id }) => {
                       {comment.created_at.slice(0, 10)}
                     </Card.Text>
                     {comment.body}
-                    <DeleteComment
-                      comment_id={comment.comment_id}
-                      setComments={setComments}
-                    ></DeleteComment>
+
+                    {userOnline.username !== comment.author ? (
+                      <Card.Footer>
+                        Cannot delete comment you not create.
```

Comment:
I think you don't need to include this anymore. Just the delete buttons for your own comments is good enough 👍🏼